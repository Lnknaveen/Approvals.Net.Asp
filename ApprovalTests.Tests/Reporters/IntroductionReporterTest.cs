using ApprovalTests.Reporters;
using NUnit.Framework;

namespace ApprovalTests.Tests.Reporters
{
	[TestFixture]
	public class IntroductionReporterTest
	{
		[Test]
		[UseReporter(typeof(ClipboardReporter))]
		public void TestComment()
		{
			Approvals.Verify(new IntroductionReporter().GetFriendlyWelcomeMessage());
		}
	}
}