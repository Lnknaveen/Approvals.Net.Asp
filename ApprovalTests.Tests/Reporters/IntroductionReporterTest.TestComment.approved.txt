Welcome to ApprovalTests.
ApprovalTests use the attribute [UseReporter(typeof(DiffReporter))] on your test class or method.
When you do this ApprovalTest will launch the result using that reporter (for example in your diff tool).
You can find several reporters in ApprovalTests.Reporters namespace, or create your own by extending ApprovalTests.Core.IApprovalFailureReporter) interface.
Find more at: http://blog.approvaltests.com/2011/12/using-reporters-in-approval-tests.html

